src/libs/state.js

```diff
@@ -11,7 +12,8 @@ export const authStore = Utils.atom({
   profile: {}
 })
 
-export const toggleStateAtom = Utils.atom({ notebooksTab: true })
+export const toggleStateAtom = staticStorageSlot(sessionStorage, 'toggleState')
+toggleStateAtom.update(v => v || { notebooksTab: true })
```

Comment:
Something about having this here feels icky, but I don't have a better idea, other than making the `slot` allocators take an optional default, which would feel less gross to me now that I think about it.

Reply:
This was happening before in `syncAtomToSessionStorage`, so it's not new, just moved.